src/utils/reshapedmsbuild.fs

```diff
@@ -131,6 +131,7 @@ module internal ToolLocationHelper =
     let visualStudioVersion120 = new Version(12, 0);
     let visualStudioVersion140 = new Version(14, 0);
     let visualStudioVersion150 = new Version(15, 0);
+    let visualStudioVersion160 = new Version(16, 0);
```

Comment:
Is this supposed to be consumed somewhere?

Reply:
Looks like lines 136 and ~760?

Reply:
Hmm, this looks very incomplete.